spec/lib/services/sku-pack-service-spec.js

```diff
@@ -227,6 +229,16 @@ describe("SKU Pack Service", function() {
                 expect(waterline.templates.destroy).to.have.been.called;
             });
         });
+
+        it('should get details on a pack', function() {
+            return skuService.getPackInfo('a')
+            .then(function(obj) {
+                expect(obj.description).to.be.a.string;
+                expect(obj.description).to.equal(data.description);
+                expect(obj.version).to.be.a.string;
+                expect(obj.version).to.equal(data.version);
+            });
+        });
```

Comment:
Test coverage problem, you forgot the negative test cases, if no description or version in the sku pack's config.json, they will be defaults to `null` as you defined.